tests/functional/vault/test_config.py

```diff
@@ -177,3 +189,11 @@ def test_vault_setGovernance(gov, vault, rando):
     # Only new governance can accept a change of governance
     with brownie.reverts():
         vault.acceptGovernance({"from": gov})
+
+
+def test_vault_setLockedProfitDegration_range(gov, vault):
+    # value must be between 0 and DEGREDATION_COEFFICIENT (inclusive)
+    vault.setLockedProfitDegration(0, {"from": gov})
+    vault.setLockedProfitDegration(DEGREDATION_COEFFICIENT, {"from": gov})
+    with brownie.reverts():
+        vault.setLockedProfitDegration(DEGREDATION_COEFFICIENT + 1000, {"from": gov})
```

Comment:
Show off-by-one behavior correct
```suggestion
        vault.setLockedProfitDegration(DEGREDATION_COEFFICIENT + 1, {"from": gov})
```